FlowCal/io.py

```diff
@@ -1375,6 +1438,9 @@ def __new__(cls, infile):
         else:
             time_step = None
 
+        # Data type
+        data_type = fcs_file.text['$DATATYPE']
+
```

Comment:
might consider using `fcs_file.text.get()` instead of accessing the dictionary via a key. this is super minor, but the error behavior is different. `.get()` will just return `None` if the key is not present I believe; accessing the dictionary directly will throw a `KeyError`. I can see cases for both, but I think we started off using `.get()` and now appear to be inconsistent with our access